src/win/tty.c

```diff
@@ -1004,6 +1004,8 @@ int uv_tty_read_start(uv_tty_t* handle, uv_alloc_cb alloc_cb,
   if (handle->tty.rd.last_key_len > 0) {
     SET_REQ_SUCCESS(&handle->read_req);
     uv_insert_pending_req(handle->loop, (uv_req_t*) &handle->read_req);
+    /* Make sure no attempt is made to insert it again until it's handled. */
+    handle->flags |= UV_HANDLE_READ_PENDING;
```

Comment:
Something tells me we should increase resq_pending here.  Otherwise the count could go below zero after the callback hits, since it will decrease it AFAIS.